client.go

```diff
@@ -732,9 +734,36 @@ func (c *Client) logStderr(r io.Reader) {
 		line, err := bufR.ReadString('\n')
 		if line != "" {
 			c.config.Stderr.Write([]byte(line))
-
 			line = strings.TrimRightFunc(line, unicode.IsSpace)
-			c.logger.Named(filepath.Base(c.config.Cmd.Path)).Debug(line)
+
+			l := c.logger.Named(filepath.Base(c.config.Cmd.Path))
+			// If output is not JSON format, print directly as error
+			if !isJSON(line) {
+				l.Debug("Error from plugin", "error", line)
```

Comment:
Log lines are not necessarily errors so I'd change this to "log from plugin" "line"